front/src/modules/trainschedule/components/ManageTrainSchedule/PowerRestrictionsSelector.tsx

```diff
@@ -8,9 +8,10 @@ import { PowerRestrictionRange } from 'applications/operationalStudies/consts';
 import { RangedValue, RollingStock } from 'common/api/osrdEditoastApi';
 import { INTERVAL_TYPES, IntervalItem } from 'common/IntervalsEditor/types';
 import IntervalsEditor from 'common/IntervalsEditor/IntervalsEditor';
-import { updatePowerRestrictionRanges } from 'reducers/osrdconf';
 import { setWarning } from 'reducers/main';
-import { getPowerRestrictionRanges } from 'reducers/osrdconf/selectors';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Reply:
Done